frontend/src/components/filters/getFilterProps.js

```diff
@@ -48,31 +48,52 @@ function getInputFilterProps(column, descriptions, filters, setFilter, setFilter
       confirm()
   }
 
-  const onToggleSwitch = checked => {
+  const onToggleSwitch = (checked, option )=> {
```

Comment:
Formatting typo around `)=>`. Also usually `(key, value)` is a more common order for those arguments.